### ISSUE TITLE
security.txt file on nodejs.org

### ISSUE THREAD

**Issue by MarcinHoppe** [nodejs/security-wg#143]
security.txt files (https://securitytxt.org/) are getting popular these days and I think they are fairly useful in reducing friction in reporting security problems.

I think putting such a file on nodejs.org Web site would not hurt. What do you think about generating and uploading such a file on the Web site?

**Comment by ChALkeR** [comment]
Yes, this needs to be done imo.
/cc @nodejs/website @nodejs/security.

**Comment by lirantal** [comment]
That's a great idea.

**Comment by bnb** [comment]
I love it. Curious how widely used `security.txt` is in production?

**Comment by ChALkeR** [comment]
@bnb That's relative new, but it's emerging pretty fast. Also, https://www.google.com/.well-known/security.txt

**Comment by Trott** [comment]
@nodejs/website

**Comment by djadmin** [comment]
+1 for security.txt. I think it could look like this -
```
Contact: security@nodejs.org
Policy: https://nodejs.org/en/security/
Contribute: https://github.com/nodejs/security-wg :)
```

Although it is supposed to be in the `/.well-known` path, it would be great to create an alias to put it in the root as well (just like robots.txt). Facebook has done it too (https://www.facebook.com/security.txt).

**Comment by MarcinHoppe** [comment]
The spec actually recommends the redirect from the root to `/.well-known`:

https://tools.ietf.org/html/draft-foudil-securitytxt-03#section-4.1

**Comment by MarcinHoppe** [comment]
PR nodejs/nodejs.org#1589 submitted against the Node.js Web site repo.

**Comment by MarcinHoppe** [comment]
The PR ☝️has been merged. We now need the PR to update NGINX configuration to be merged: nodejs/build#1181.

**Comment by MarcinHoppe** [comment]
Both PRs have been merged, the file has been up on the site for some time now and all seems to be working fine. Closing.
